<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snap_points_widget_t</class>
 <widget class="QWidget" name="snap_points_widget_t">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>41</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btn_toggle_snap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Toggle Snapping</string>
     </property>
     <property name="text">
      <string>S</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_snap_center">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Snap to Center</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_snap_verts">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Snap to Vertex</string>
     </property>
     <property name="text">
      <string>V</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_snap_edge_midpoint">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Snap to Edge Midpoint</string>
     </property>
     <property name="text">
      <string>E</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sld_snap_threshold">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Snap Threshold</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dsb_snap_threshold">
     <property name="toolTip">
      <string>Snap Threshold</string>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
